server-test/src/test/scala/testpkg/ClientTest.scala

```diff
@@ -87,32 +87,38 @@ object ClientTest extends AbstractServerTest {
     assert(client("compile;willFail;willSucceed") == 1)
   }
   test("compi completions") { _ =>
-    val expected = Vector(
+    val expected = Set(
```

Comment:
Could you switch this back to Vector? Part of the spec for client completions is that the outputs be returned in sorted order.

Reply:
If you want to merge without re-running CI, can switch back to Vector in one of my outstanding branches.